transformer_engine/pytorch/module/linear.py

```diff
@@ -641,6 +641,40 @@ def __init__(
         else:
             self.gemm_bias_unfused_add = False
 
+    def get_fp8_weights_empty_tensors(self) -> list:
```

Comment:
This should probably be moved to the base class. That would allow adding this functionality to the other granular APIs.

Reply:
For FP8 case this doesn't return anything, bug?

Reply:
Right, fixed now